### ISSUE TITLE
Do not show image link previews for descriptions containing images

### ISSUE THREAD

**Issue by brennan3** [delphidigital/delphi-tg-bots#31]
ie, https://twitter.com/26shirts/status/1785383301642895604

**Comment by SherbetLemon47** [comment]
Can you clarify what exactly needs to be done here?

**Comment by brennan3** [comment]
There is no bounty on the issue at this point in time. If you are still interested in working on it, lmk and I'll add more details :)

**Comment by SherbetLemon47** [comment]
Yeah! Ill still try to solve it! I just didn't get what needs to be done
